conda-recipe/conda_build_config.yaml

```diff
@@ -15,4 +15,8 @@
 #===============================================================================
 
 numpy:
-  - 1.19
+  - 1.24

```

Comment:
And actually on second thought: since these are **build-time** dependencies rather than runtime, perhaps would be better to switch them to numpy>=2, since a module built with numpy>=2 is usable with a runtime numpy<2, but not the other way around.

Reply:
I would like to leave numpy installation behavior similar to usual testing: https://github.com/intel/scikit-learn-intelex/pull/1992#discussion_r1804754817

Reply:
@Alexsandruss Wouldn't that leave parts of the conda package broken if a user first install scikit-learn-intelex and then later on upgrades numpy to a version >= 2?